### PR TITLE
Homework2: Danila Zhogol

### PR DESCRIPTION
**Pull request by danzhog**

### CODE REVIEW

File: src/extratask1.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# Определите, является ли число палиндромом
+# (читается слева направо и справа налево одинаково)
+# Число положительное целое, произвольной длины.
+# Задача требует работать только с числами (без конвертации числа в строку)
+
+a = int(input('введите число: '))
+
+b = a  # дублируем число, чтобы сохранить изначальное
```
Comment (belovmd) [inline src/extratask1.py]: you don't need to comment every line

Comment (belovmd) [inline src/extratask1.py]: unclear naming

File: src/extratask2.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+n = int(input('сколько первых чисел Фибоначчи вы хотите увидеть: '))
+
+a1 = 0
+a2 = 1
+c = 0
+
+for i in range(n):
+    print(a1)
+    c = a1 + a2
```
Comment (belovmd) [inline src/extratask2.py]: a1, a2 = a2, a1+a2

File: src/extratask2.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
+# Выведите n-ое число Фибоначчи, используя только временные переменные,
+# циклические операторы и условные операторы. n - вводится
+
+n = int(input('сколько первых чисел Фибоначчи вы хотите увидеть: '))
+
+a1 = 0
```
Comment (belovmd) [inline src/extratask2.py]: unclear naming

File: src/task2.py [hunk 4]
```diff
@@ -0,0 +1,56 @@
+from string import Template
+#  Зарегистрируйтесь на одном (или нескольких) из сайтов:
+# https://py.checkio.org/ , https://www.codewars.com,
+#  https://www.hackerrank.com/, https://acmp.ru
+# И решите 1-5 задач уровня Elementary или любых других.
+
+# mission1 on py.checkio.org
+
+
```
Comment (belovmd) [inline src/task2.py:15]: please add tasks descriptions

File: src/task3.py [hunk 5]
```diff
@@ -0,0 +1,29 @@
+# Найти самое длинное слово в введенном предложении.
+# Учтите что в предложении есть знаки препинания.
+
+s1 = str(input('введите предложение: '))
+b = s1.split()  # делим предложение на 'массив' слов
+# (к концу слов всё еще прикреплены знаки препинания)
+
+for i in range(len(b)):  # удаляем у каждого слова
+    # 'прилепленные' знаки препинания
+    b[i] = b[i].strip(",.!*?='")
```
Comment (belovmd) [inline src/task3.py]: what about " []{}() ?

File: src/task3.py [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+
+max = b[0]  # временная переменная хранит самое длинное слово на этот момент
+lst = []
+
+for i in range(len(b)):  # сравниваем все слова начиная с первого
```
Comment (belovmd) [inline src/task3.py]: for word in b:

File: src/task3.py [hunk 7]
```diff
@@ -0,0 +1,29 @@
+# Найти самое длинное слово в введенном предложении.
+# Учтите что в предложении есть знаки препинания.
+
+s1 = str(input('введите предложение: '))
+b = s1.split()  # делим предложение на 'массив' слов
```
Comment (belovmd) [inline src/task3.py]: bad naming

**Review by belovmd — COMMENTED** [review]
1. please don't remove init file
2. external tasks are pretty easy you can try to solve some advanced tasks if you want
3. Please address these issues

File: src/extratask2.py (hunk 3 above)
Comment (danzhog) [reply src/extratask2.py]: done

File: src/extratask2.py (hunk 2 above)
Comment (danzhog) [reply src/extratask2.py]: done

File: src/extratask1.py (hunk 1 above)
Comment (danzhog) [reply src/extratask1.py]: done

Comment (danzhog) [reply src/extratask1.py]: done

File: src/task3.py (hunk 5 above)
Comment (danzhog) [reply src/task3.py]: done

File: src/task3.py (hunk 7 above)
Comment (danzhog) [reply src/task3.py]: done

File: src/task3.py (hunk 6 above)
Comment (danzhog) [reply src/task3.py]: "i" is changed to "word"
but syntax word in b not working in my situation

File: src/task2.py (hunk 4 above)
Comment (danzhog) [reply src/task2.py:15]: done

**Review by belovmd — COMMENTED** [review]
7,5

**Pull request closed without merging**